local_db/data_access.py

```diff
@@ -128,6 +128,14 @@ def get_requests_authored_by_user(self, username: str):
             )
         ]
 
+    def get_requests_for_workspace(self, workspace: str):
+        return [
+            self._request(request)
+            for request in RequestMetadata.objects.filter(workspace=workspace).order_by(
+                "status"
```

Comment:
Would ordering by date be more useful?

Reply:
probably. I was not worrying too much about this list on the basis that it would be short for now & probably need more work later to make it useful if/when it gets long (i.e. pagination)

Reply:
we can order by created_at - I would prefer to order by updated_at but I don't want to have to calculate that! https://github.com/opensafely-core/airlock/blob/main/local_db/models.py#L80-L90